frontend/packages/shared/src/utils/featureToggleUtils.test.ts

```diff
@@ -46,6 +46,22 @@ describe('featureToggle url', () => {
     window.history.pushState({}, 'PageUrl', '/');
     expect(shouldDisplayFeature('shouldOverrideAppLibCheck')).toBe(false);
   });
+
+  it('should persist features in sessionStorage when persistFeatureFlag is set in url', () => {
+    window.history.pushState(
+      {},
+      'PageUrl',
+      '/?featureFlags=customizeEndEvent,shouldOverrideAppLibCheck&persistFeatureFlag=true',
+    );
+    expect(shouldDisplayFeature('componentConfigBeta')).toBe(false);
+    expect(shouldDisplayFeature('shouldOverrideAppLibCheck')).toBe(true);
+    expect(shouldDisplayFeature('customizeEndEvent')).toBe(true);
+    expect(typedSessionStorage.getItem<string[]>('featureFlags')).toEqual([
+      'shouldOverrideAppLibCheck',
+      'customizeEndEvent',
+    ]);
+    expect(typedLocalStorage.getItem<string[]>('featureFlags')).toBeUndefined();
```

Comment:
We should ensure to clean up all local storage between tests, to avoid conflicts like we did here. 😊 

```